packages/@okta/vuepress-site/docs/guides/sign-users-out/main/springboot/addbaseuri.md

```diff
@@ -1,2 +1 @@
-Spring Boot provides support for logout handling right out of the box. Most of the defaults for the security components can be easily configured.
-See [Spring Security Logout Handling](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#jc-logout) for insights on this topic.
+Spring Boot supports sign-out handling right out of the box. Most of the defaults for the security components can be easily configured. See [Spring Security Logout Handling](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#jc-logout).
```

Comment:
`out of the box` seems a little colloquial. What if we adjust it While we're here?

```suggestion
Spring Boot supports sign-out handling by default. Most of the security components default settings can be easily configured. See [Spring Security Logout Handling](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#jc-logout).
```

Reply:
Seems good - fixed